tests/Unit/Domain/CoordinateMaps/Test_Wedge3D.cpp

```diff
@@ -268,39 +301,24 @@ void test_wedge3d_fail() noexcept {
                           test_mapped_point7);
   }
 }
-
 }  // namespace
 
 SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge3D.Fail", "[Domain][Unit]") {
   test_wedge3d_fail();
 }
 
-SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge3D.Map.Equiangular",
-                  "[Domain][Unit]") {
-  test_wedge3d_all_directions(true);
+SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge3D.Map", "[Domain][Unit]") {
```

Comment:
Could you combine the Alignment, Map, RandomRadii tests into a single test case that just calls the different functions?